mds/db/load.py

```diff
@@ -102,6 +102,11 @@ def load_from_df(self, df, record_type, table, before_load=None, stage_first=Tru
                 if query is not None:
                     conn.execute(query)
 
+            # Delete the tmptable  since we did fake tmp tables
+            # and not using Postgres TEMPORARY 
+            conn.execute(f"DROP TABLE {temp}")
```

Comment:
I think `conn` is not in scope at this point, since it's outside the `with`, right?

What do you think of moving this directly under the other `conn.execute` (line 103)?

Reply:
that is fine with me I think.